file.go

```diff
@@ -5,6 +5,7 @@ type FileKind string
 
 const (
 	FileKindWhiteout = FileKind("whiteout")
+	FileKindRPM      = FileKind("rpm")
```

Comment:
So every file's "kind" is effectively the component that noticed it?

Reply:
I suppose in theory any filescanner can create any `FileKind` and any `Resolver` can be interested in any `FileType` or perhaps multiple in the future, so the name is kind of arbitrary. Do you think  the kind could be better named?

Reply:
I guess not? It's fine.